v21.2/transactions.md

```diff
@@ -163,6 +163,8 @@ Transactions in CockroachDB lock data resources that are written during their ex
 
 For more details about transaction contention and best practices for avoiding contention, see [Understanding and Avoiding Transaction Contention](performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention).
 
+For further background on database contention, see [What is Database Contention, and Why Should You Care?](https://www.cockroachlabs.com/blog/what-is-database-contention/).
```

Comment:
One more nit: Since these references are fairly similar, perhaps we should combine into something like this?

> For more details about transaction contention and best practices for avoiding contention, see [What is Database Contention, and Why Should You Care?](https://www.cockroachlabs.com/blog/what-is-database-contention/) and [Understanding and Avoiding Transaction Contention](performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention).

Reply:
I like that idea better. Will update shortly.